ui/app/adapters/pki/tidy.js

```diff
@@ -33,4 +34,13 @@ export default class PkiTidyAdapter extends ApplicationAdapter {
     const url = this.urlForCreateRecord(snapshot);
     return this.ajax(url, 'POST', { data: this.serialize(snapshot) });
   }
+
+  queryRecord(store, type, query) {
```

Comment:
originally I had `findRecord` here so that the backend would be the models id, however we can have two possible models for tidying. A manual tidy will be created for performing a manual operation (will only ever be created to make a POST request). There will always be an auto tidy model, that we can read an edit
```js

  findRecord(store, type, backend) {
    // only auto-tidy will ever be read, no need to pass the type here
    return this.ajax(`${this._baseUrl(backend)}/config/auto-tidy`, 'GET').then((resp) => {
      return resp.data;
    });
  }
```

thoughts on handling it this way? so the `tidyType` will set the id on the model as either "auto" or "manual". Open to naming suggestions too